src/apps/uclient/uclient.c

```diff
@@ -34,12 +34,11 @@
 #include "ns_turn_utils.h"
 #include "session.h"
 
+#if defined(__linux__)
```

Comment:
Same as above (`__linux__` vs `__unix_`)

Reply:
See https://github.com/coturn/coturn/issues/1044 .
After collecting everyone's needs, we will modify them later in a separate PR.